ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/InitializeMigrationCommand.java

```diff
@@ -0,0 +1,134 @@
+
+@Command(
+    name = "initialize",
+    description = "Initializes the schema metadata (stream and table)."
+)
+public class InitializeMigrationCommand extends BaseCommand {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(InitializeMigrationCommand.class);
+
+  private String createEventStream(final String name, final String topic, final int replicas) {
+    return "CREATE STREAM " + name + " (\n"
+        + "  version_key  STRING KEY,\n"
+        + "  version      STRING,\n"
+        + "  name         STRING,\n"
+        + "  state        STRING,  \n"
+        + "  checksum     STRING,\n"
+        + "  started_on   STRING,\n"
+        + "  completed_on STRING,\n"
+        + "  previous     STRING\n"
+        + ") WITH (  \n"
+        + "  KAFKA_TOPIC='" + topic + "',\n"
+        + "  VALUE_FORMAT='JSON',\n"
+        + "  PARTITIONS=1,\n"
+        + "  REPLICAS= " + replicas + " \n"
+        + ");\n";
+  }
+
+  private String createVersionTable(final String name, final String topic) {
+    return "CREATE TABLE " + name + "\n"
+        + "  WITH (\n"
+        + "    KAFKA_TOPIC='" + topic + "'\n"
+        + "  )\n"
+        + "  AS SELECT \n"
+        + "    version_key, \n"
+        + "    latest_by_offset(version) as version, \n"
+        + "    latest_by_offset(name) AS name, \n"
+        + "    latest_by_offset(state) AS state,     \n"
+        + "    latest_by_offset(checksum) AS checksum, \n"
+        + "    latest_by_offset(started_on) AS started_on, \n"
+        + "    latest_by_offset(completed_on) AS completed_on, \n"
+        + "    latest_by_offset(previous) AS previous\n"
+        + "  FROM migration_events \n"
+        + "  GROUP BY version_key;\n";
+  }
+
+  @Override
+  public void run() {
+    final long startTime = System.currentTimeMillis();
+
+    final MigrationConfig properties = MigrationConfig.load();
+    final String streamName = properties.getString(MigrationConfig.KSQL_MIGRATIONS_STREAM_NAME);
+    final String tableName = properties.getString(MigrationConfig.KSQL_MIGRATIONS_TABLE_NAME);
+    final String eventStreamCommand = createEventStream(
+        streamName,
+        properties.getString(MigrationConfig.KSQL_MIGRATIONS_STREAM_TOPIC_NAME),
+        properties.getInt(MigrationConfig.KSQL_MIGRATIONS_TOPIC_REPLICAS)
+    );
+    final String versionTableCommand = createVersionTable(
+        tableName,
+        properties.getString(MigrationConfig.KSQL_MIGRATIONS_TABLE_TOPIC_NAME)
+    );
+    final String ksqlServerUrl = properties.getString(MigrationConfig.KSQL_SERVER_URL);
+
+    final URL url;
+    try {
+      url = new URL(ksqlServerUrl);
+    } catch (MalformedURLException e) {
+      LOGGER.error("Invalid ksql server URL: " + ksqlServerUrl);
+      System.exit(1);
+      return;
+    }
+
+    final ClientOptions options = ClientOptions
+        .create()
+        .setHost(url.getHost())
+        .setPort(url.getPort());
+
+    final Client ksqlClient = Client.create(options);
```

Comment:
Creating the Java client from the MigrationsConfig should be factored out into a util file, since many more commands will need this. Feel free to defer to a follow-up PR.